docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -117,6 +117,141 @@ The values represent the speed and direction to command the motor to, normalized
 Refer to the `uavcan.equipment.esc.RawCommand section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#rawcommand>`_ in the specification 
 for more details
 
+uavcan.tunnel.Broadcast (Data Type ID = 2010)
```

Comment:
I chose to add this to where we already talk about all of the supported messages. This one is slightly unique because it has an example with it